config_builder.go

```diff
@@ -423,6 +423,13 @@ func (d *DefaultWalletImpl) BuildWalletConfig(ctx context.Context,
 			return nil, nil, nil, err
 		}
 
+		// If we have a macaroon root key from the init wallet params,
+		// set the root key before baking any macaroons
+		if len(walletInitParams.MacRootKey) > 0 {
+			err := macaroonService.SetRootKey(walletInitParams.MacRootKey)
+			return nil, nil, nil, err
```

Comment:
This return should probably be wrapped by a `if err != nil {` condition?

Reply:
Thanks for the catch, this was a lint error I addressed right before committing (previously I was ignoring the return value of `SetRootKey` and I added the `err :=` and `return` but forgot the `if`.